bsp/imx/imx8mmevk.cfg

```diff
@@ -223,3 +240,25 @@ CONFIG_CRYPTO_DEV_FSL_CAAM_SECVIO=y
 CONFIG_CRYPTO_DEV_FSL_CAAM=y
 CONFIG_CRYPTO_DEV_FSL_CAAM_SM=y
 CONFIG_CRYPTO_DEV_FSL_CAAM_SM_TEST=m
+CONFIG_IMX_DSP_REMOTEPROC=m
+CONFIG_CLK_VEXPRESS_OSC=y
+CONFIG_COMMON_CLK_SCMI=y
+CONFIG_RPMSG_CHAR=m
+CONFIG_ARM_IMX_BUS_DEVFREQ=m
+CONFIG_ARM_IMX8M_DDRC_DEVFREQ=m
+CONFIG_INTERCONNECT_IMX=m
+CONFIG_INTERCONNECT_IMX8MQ=m
+CONFIG_CRYPTO_BLAKE2S=m
+CONFIG_CRYPTO_USER_API_HASH=m
+CONFIG_CRYPTO_USER_API_SKCIPHER=m
+CONFIG_CRYPTO_DEV_HISI_TRNG=m
+CONFIG_CRC8=y
+CONFIG_DEBUG_INFO_REDUCED=y
+
+# enable KTLS
+CONFIG_TLS=y
+CONFIG_TLS_DEVICE=y
+
+# MGS-6358
+CONFIG_CGROUP_FREEZER=y
```

Comment:
This is already provided by cgroups.

features/cgroups/cgroups.cfg:CONFIG_CGROUP_FREEZER=y